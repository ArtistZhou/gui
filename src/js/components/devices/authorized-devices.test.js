import React from 'react';
import renderer from 'react-test-renderer';
import Authorized from './authorized-devices';

it('renders correctly', () => {
  const tree = renderer.create(<Authorized devices={[]} />).toJSON();
  expect(tree).toMatchSnapshot();
});
