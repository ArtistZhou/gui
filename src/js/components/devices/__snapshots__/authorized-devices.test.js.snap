// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="relative"
>
  <div
    className="loaderContainer shrunk"
  >
    <div
      className="loader"
    >
      <span
        className="dot dot_1"
      />
      <span
        className="dot dot_2"
      />
      <span
        className="dot dot_3"
      />
      <span
        className="dot dot_4"
      />
    </div>
  </div>
  <div
    className="dashboard-placeholder"
  >
    <p>
      No devices found
    </p>
    <p>
      No devices have been authorized to connect to the Mender server yet.
    </p>
  </div>
  <div />
</div>
`;
